tests/e2e/scenarios/test_reborn_webui_v2_legacy_settings_search.py

```
@@ -324,6 +324,16 @@ def _provider_card(page, provider_id: str):
     )
 
 
+async def _choose_select_menu_option(root, label: str, option: str):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_choose_select_menu_option`

Add return type annotation: `None`

(ANN202)